http-kit/src/io/pedestal/http/http_kit.clj

```diff
@@ -136,3 +139,82 @@
           ;; the test contract (nil or InputStream).
           (update response :body test/convert-response-body))))))
 
+(defn- is-byte-array?
```

Comment:
Why not `bytes?` what is the difference? It would be nice to leave a doc justifying the reason for not using the core

Reply:
No, I can change that. I never noticed `bytes?` before.